.github/dependabot.yml

```
@@ -10,5 +10,21 @@ updates:
     # Check the npm registry for updates every month
     schedule:
       interval: "monthly"
-    # Bump versions only in package-lock.json
-    versioning-strategy: "lockfile-only"
+    # Usually only bump versions in package-lock.json, but update package.json for major version updates
+    versioning-strategy: "increase-if-necessary"
+    # Update all dependencies in a single PR
+    groups:
+      # ESLint usually requires updating together for major updates
+      eslint:
+        patterns:
+          - "eslint*"
+          - "@typescript-eslint/*"
```

Looking at [what will be needed for TypeScript ESLint in the future](https://typescript-eslint.io/getting-started#quickstart), we may want to already add `@eslint/*` too:
```suggestion
      eslint:
        patterns:
          - "eslint*"
          - "@eslint/*"
          - "@typescript-eslint/*"
```